data_structure.cpp

```diff
@@ -50,9 +50,10 @@ double DocumentShippingForm::getShippingPrice(Price custom_price) {
 	return (distance * custom_price.DOC_distance + custom_price.DOC_service);
 }
 
-void DocumentShippingForm::inputInfo() {
+void DocumentShippingForm::inputInfo(std::ifstream &filein) {
 	std::cout << "Enter distance (km): ";
-	std::cin >> distance;
+	filein>>distance;
+	//std::cin >> distance;
```

Comment:
Cậu check lại toàn bộ code, chỗ nào thấy không cần comment thì xóa hết nhé. VD dòng 56 bỏ đi cho gọn

Khi đẩy lên Pull Request thì chỉ comment nếu cần mô tả hoặc giải thích code thôi